app/code/Bulbulatory/Recommendations/Setup/Patch/Data/InsertRecommendations.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Setup\Patch\Data;
+
+use Bulbulatory\Recommendations\Setup\Patch\Schema\UpgradeHashRecommendations;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+class InsertRecommendations implements DataPatchInterface
+{
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup
+    ) {
+
+        $this->moduleDataSetup = $moduleDataSetup;
+    }
+
+    public function apply()
+    {
+        $this->moduleDataSetup->startSetup();
+        $setup = $this->moduleDataSetup;
+
+        $data = [
+            ['reference' => 'Very good site', 'email' => 'test_one@test_one.com'],

```

Comment:
`reference` (customer reference) is not meant to be a message, it's meant to be a reference to an existing customer - if a recommendation is sent, we have to know who sent it.

Reply:
@TM18

> reference (customer reference) is not meant to be a message, it's meant to be a reference to an existing customer - if a recommendation is sent, we have to know who sent it.

Thank you! I misunderstood this field description. I fixed this problem